java/monkey/src/main/java/root/lexer/Lexer.java

```diff
@@ -69,7 +74,7 @@ private void advance() {
         this.pos++;
     }
 
-    private char getCc() {
+    char getCc() {
```

Comment:
We can swap this to be `protected` at some point in the future. Since `LexerIterable` is in the same package.

Reply:
Changed to protected.